src/main/java/sirius/biz/tenants/TenantUserManager.java

```diff
@@ -684,7 +686,7 @@ protected boolean checkApiToken(LoginData loginData, String givenApiToken) {
         }
 
         long currentTimestampInDays = TimeUnit.MILLISECONDS.toDays(System.currentTimeMillis());
-        // Timestamps of tomorrow and yesterday should be valid too, to be more gracefull with nightly scripts utilizing
+        // Timestamps of tomorrow and yesterday should be valid too, to be more gracefully with nightly scripts utilizing
```

Comment:
Not sure, but I believe it should be _graceful_, as it is probably no adverb here. It does not describe "the art of being", but rather "the result of being".

Reply:
_graceful_ sounds better to me, too.